dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2509,6 +2537,15 @@
         }
       }
     },
+    "DocumentDetectedLanguage": {
+      "type": "object",
+      "properties": {
+        "detectedLanguage": {
+          "type": "string",
+          "description": "If 'language' is set to 'auto' for the document in the request this field will contain a 2 letter ISO 639-1 representation of the language detected for this document."
```

Comment:
Is this the same as BCP-47? So far, everything I've been reviewing for Cognitive Services has always mentioned BC-47.

Reply:
Did a quick look-up and it seems to me they are very similar. However, I'd ask @assafi to comment on this.

Reply:
BCP-47 is a superset of ISO 639. We need to use ISO here because that's what is currently supported by LD.

We agreed to replace this string with the LD result object https://github.com/Azure/azure-rest-api-specs/blob/b1f713e352f4f0316dfe87f94127a579fedb4fab/dev/cognitiveservices/data-plane/Language/analyzetext.json#L2233

@nourhanFarid heads up to update the service.

Reply:
@heaths I agreed with Assaf to postpone this change to October APIs (2022-10-01-preview) as we do not have bandwidth to this change before July CC. Can you please approve and help us merge this whenever you have time?